tail_test.go

```diff
@@ -95,308 +95,310 @@ func TestStop(t *testing.T) {
 	tail.Cleanup()
 }
 
-func TestStopAtEOF(_t *testing.T) {
-	t := NewTailTest("maxlinesize", _t)
-	t.CreateFile("test.txt", "hello\nthere\nworld\n")
-	tail := t.StartTail("test.txt", Config{Follow: true, Location: nil})
+func TestStopAtEOF(t *testing.T) {
+	tailTest := NewTailTest("maxlinesize", t)
+	tailTest.CreateFile("test.txt", "hello\nthere\nworld\n")
+	tail := tailTest.StartTail("test.txt", Config{Follow: true, Location: nil})
 
 	// read "hello"
-	<-tail.Lines
+	line := <-tail.Lines
+	if line.Text != "hello" {
+		t.Errorf("Expected to get 'hello', got '%s' instead", line)
```

Comment:
hmmm, It will output the object like `'&{hello 2016-04-26 16:27:38.9467369 -0700 PDT <nil>}'`. Needs to be fixed.